client/stylesheets/_modal.scss

```diff
@@ -1136,7 +1136,45 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 /*~~ Join Button End ~~*/
 
 .modJoinWrapper {
+	width: 70% !important;
+	margin: 60px auto;
+}
+
+.modJoinWrapper h1 {
+	font-size: 2.4rem;
+	font-weight: 400;
+	margin-bottom: 0;
+}
+
+.modJoinWrapper ul li {
+	line-height: 2;
+}
+
+.modJoinOnly {
+	margin-top: 20px;
+	font-size: 1.4rem !important;
+}
+
+.modJoinFB-Btn {
+	margin-top: 20px;
+	background-color: #3A5795 !important;
+	border-radius: 4px;
+	color: #fff !important;
+	width: 40%;
+	padding: 10px;
+	text-align: center;
+}
+
+.modJoinFB-Btn b {
+	font-weight: 400 !important;
+	font-size: 1rem !important;
+}
 
+.modJoinFB-Btn i {
```

Comment:
Merge rule `.modJoinFB-Btn i` with rule on line 1158<br>Selector `modJoinFB-Btn` should be written in lowercase with hyphens